src/components/TeamMember/TeamMember.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
 
 const TeamMember = ({ info }: Props) => (
   <Link
-    href={`/user/${info.userId}`}
+    href={`/user/${info.id as unknown as string}`}
```

Comment:
이 부분 저도 타입 에러 떠서 이런 식으로 수정했던 부분이 많은 것 같은데 그냥 아래처럼 info.id로 작성해도 될 것 같아요! 에러 뜰 때는 any 관련 에러가 뜨는데 사실 info는 MemberInfo 타입으로 설정되어 있고, MemberInfo에서 id는 number로 타입 지정했기 때문에 에러가 안뜨는 게 맞는 것 같슴다..! 저희 노트북에서 에러가 뜨는 이유는 노트북이 느리거나, 타입스크립트 컴파일러가 느리기 때문이지 않을까... 추측해봅니다!
```suggestion
    href={`/user/${info.id}`}
```

Reply:
저렇게 수정하고 싶지만....수정하면 커밋 자체가 안돼요 정기님ㅠㅠ흑흑

Reply:
아하 그렇군요ㅠㅠ